torch/csrc/distributed/c10d/FlightRecorder.hpp

```diff
@@ -24,9 +24,10 @@ namespace c10d {
 // (minor when adding fields, major when changing existing fields)
 // Also update both JSON and Pickle dumps to make use of the newly defined
 // field(s).
-DEFINE_CONSTANT(version_val, "2.5")
+DEFINE_CONSTANT(version_val, "2.6")
```

Comment:
Maybe use [`ncclGetVersion`](https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/api/comms.html#ncclgetversion) rather than defining a macro/constant? (reduce maintenance load)